e2e-tests/benchmark.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+set -Eexuo pipefail
+SCRIPT_DIR="$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+# Add root directory to the PATH
+export PATH="$SCRIPT_DIR/..:$PATH"
+
+ROOT_DIR="$SCRIPT_DIR/.."
+
+cd "$ROOT_DIR"
+
+# Remove downloaded didc, drun.
+trap 'rm $ROOT_DIR/didc $ROOT_DIR/drun' EXIT SIGINT
+
+get_didc_release(){
```

Comment:
I think almost all this logic belongs better in the `run-benchmark.sh` script. Otherwise, running `cargo bench` outside this script doesn't work.

Reply:
Done.